docs/usage.md

```diff
@@ -222,24 +222,24 @@ Otherwise, you can provide your own position file in the `--steps impute` with S
 nextflow run nf-core/phaseimpute --input samplesheet.csv --steps impute --posfile samplesheet_posfile.csv  --tool stitch --outdir results --genome GRCh37 -profile docker
 ```
 
-The csv provided in `--posfile` must contain two columns [chr, file]. The first column is the chromosome and the file column are tsvs with the list of positions, unique to each chromosome.
+The csv provided in `--posfile` must contain four columns [panel, chr, vcf, txt]. The first column [panel] is a name to identify the sites, typically a panel name. The second column [chr] is the chromosome corresponding to that file. The third column [vcf] can be kept empty. Lastly, the fourth column [txt] is a compressed tsv file containing the list of positions, unique to each chromosome.
 
 ```console
-chr,file
-chr1,posfile_chr1.txt
-chr2,posfile_chr2.txt
-chr3,posfile_chr3.txt
+panel,chr,vcf,txt
+1000G,chr1,,posfile_chr1.tsv.gz
+1000G,chr2,,posfile_chr2.tsv.gz
+1000G,chr3,,posfile_chr3.tsv.gz
 ```
 
-The file column should contain a TSV with the following structure, from STITCH documentation: "File is tab separated with no header, one row per SNP, with col 1 = chromosome, col 2 = physical position (sorted from smallest to largest), col 3 = reference base, col 4 = alternate base. Bases are capitalized. STITCH only handles bi-allelic SNPs" [STITCH](https://github.com/rwdavies/STITCH/blob/master/Options.md).
+The fourth column containing the compressed file has a TSV with the following structure, similar to that from STITCH documentation: "File is tab separated with no header, one row per SNP, with col 1 = chromosome, col 2 = physical position (sorted from smallest to largest), col 3 = reference base, col 4 = alternate base. Bases are capitalized. STITCH only handles bi-allelic SNPs" [STITCH](https://github.com/rwdavies/STITCH/blob/master/Options.md). Unlike the files used in the original STITCH program, in `phaseimpute`, the last column, containing the reference vs. the alternate base is comma-separated.
```

Comment:
I think the `Stitch` format is more straight forward.
This would simplify how the user implement it, no ?
If you want to keep to `glimpse` format, maybe just specify `col 3 = reference,alternate base` ?

Reply:
We should maybe give the `bcftools` command to generate this file ?

Reply:
I agree that the stitch format is more straightforward, but at the same time we would have to re-transform the file if we want to use glimpse1. At the moment, we could stick with the glimpse1 format and then think of a more elegant solution.